remix-tests/src/runTestSources.ts

```diff
@@ -33,7 +45,12 @@ export async function runTestSources(contractSources, versionUrl, usingWorker, t
         function compile (next) {
             compileContractSources(contractSources, versionUrl, usingWorker, importFileCb, { accounts }, next)
         },
-        function deployAllContracts (compilationResult, next) {
+        function deployAllContracts (compilationResult, asts, next) {
+            for(const filename in asts)
+            {
```

Comment:
We use the Javascript style for bracket for loops. I'm fine using the C++ version, but if it doesn't make much difference for you I would recommend doing it like that : 
```typescript
for (const filename in asts) {  // Bracket at the end of the line
  if(filename.includes('_test.sol')) sourceASTs[filename] = asts[filename].ast
}
```

Reply:
obviously, we should use javascript style